src/Controllers/ImageController.php

```diff
@@ -72,6 +72,46 @@ public function __invoke(Request $request, int $store, string $size, string $pla
         return $this->storage()->response($resizedPath);
     }
 
+    public function getResizedPath(string $size, string $file, string $webp)
+    {
+        foreach (config('imageresizer.external') as $placeholder => $url) {
+            if (Str::startsWith($file, $placeholder)) {
+                $file = Str::replaceFirst($placeholder, '', $file);
+                $placeholderUrl = $url;
+                break;
+            }
+        }
+
+        $placeholder = isset($placeholderUrl)
+            ? $placeholder
+            : 'local';
+
+        return config('rapidez.store').'/resizes/'.$placeholder.'/'.$size.'/'.$file.$webp;
+    }
+
+    public function redirectFromSku(Request $request, string $size, string $file)
+    {
+        $webp = str_ends_with($file, '.webp') ? '.webp' : '';
+        $baseFile = $webp ? str_replace_last('.webp', '', $file) : $file;
+
+        $sku = pathinfo($baseFile)['filename'];
+        $file = $this->productImageUrlFromSku($sku);
+        return redirect($this->getResizedPath($size, $file, $webp), 301);
```

Comment:
Is there ever a chance that the url is not from Magento?
I suggest using the named route instead of building it yourself
https://github.com/rapidez/image-resizer/pull/24/files#r1231931437
```suggestion
        return redirect(route('resized-image', ['placeholder' => 'magento', ...compact('size', 'file', 'webp', 'store')]), 301);
```

This way you don't need to add `magento` to the productImageUrl either